### PR TITLE
adding 1.10 golang support

### PR DESCRIPTION
**Pull request by Jimexist**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: go
 go:
   - 1.8
   - 1.9
+  - "1.10"
```
Comment (ddysher) [inline .travis.yml]: why quote for 1.10 but not others?

**Comment by ddysher** [comment]
@Jimexist what's the status?

**Comment by Jimexist** [comment]
@ddysher blocked because golang 1.10 format code differently than other versions.

my suggestion is to remove the `gofmt` check in `hack`

**Comment by ddysher** [comment]
looks like people are having similar problem https://gitlab.cncf.ci/coredns/coredns/commit/05e41c59698cadb6c43452504e4a999570ab93b0

**Comment by kdada** [comment]
Close due to #205

**Pull request closed without merging**